packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -273,6 +281,58 @@ You need to update your Authorization Server to use your custom domain to fix th
+
+## Configure a custom email domain
+
+1. In the Admin Console, go to **Settings** and then **Emails & SMS**.
+
+2. Click the **Sender:** link (**Okta <noreply@okta.com>**) near the top of the page.
+
+3. On the Configure Email Sender dialog box, select **Custom email domain** as the type of sender that you want to send system notification emails from.
+
+4. In the **Email address to send from**, enter the email address that you want to send the system notification emails from. This is what displays in the emails sent to your users.
+
+5. Enter the **Name of sender**. This name appears as the sender in the emails sent to your users.
+
+6. In the **Mail domain to send from** box, enter a unique mail domain that your organization has dedicated for Okta to send mail from. Later in this procedure, you add the unique mail domain to the SPF record to your DNS zone (the root domain) as an include-statement to show that you allow Okta to send mail from this unique mail domain.
+
+7. Click the **Save & View Required DNS Records** button to save your changes and view your org's DNS records that you need to update before your settings can take effect.
+
+8. Update your DNS records using the provided values.
+
+9. When you have updated your DNS records through your domain provider, click **I've updated the DNS records**. Okta begins polling your DNS records until it detects your updates (may take up to 24 hours). Your configuration is pending until the DNS updates are detected.
+
+    Alternatively, you can click **I will update the DNS records later**. Your records aren't polled and your configuration is incomplete until you update the relevant DNS records and click **I've updated the DNS records**. You can view the list of records that require an update at any time.
+
+10. Add the SPF record to your DNS zone (the root domain). An SPF record specifies the mail servers that your organization has authorized to send mail from your domain. If your root domain already has an SPF record, the following update can prevent spoofers from sending mail that mimics your domain.
+
+    For example, if you only send mail from Microsoft Office 365, your SPF record has an include-statement like this:
+
+    ```plain
+    example.com TXT    v=spf1 include:spf.protection.outlook.com -all
+    ```
+
+    To finish configuring your custom email domain, you must add another include-statement that specifies the mail domain that you specified in the **Mail domain to send from** box in step 6. This is also the host that appears in the first CNAME row in the DNS Records table in the Configure Email Sender dialog box.
+
+    ![CNAME Example](/img/CNAMEExample.png "CNAME table with an arrow pointing at the first CNAME row in the table")
+
+    Add the host to the existing record to configure a combined SPF record similar to this:
+
+    ```plain
+    example.com TXT    v=spf1 include:oktamail.example.com include:spf.protection.outlook.com -all
+    ```
```

Comment:
Not sure if this is a typo, or it's supposed to be like this, but the diagram has host="oktaemail.example.com", but the SPF record has "oktamail.example.com" ?? Who can we confirm with?